pkg/git/git.go

```diff
@@ -658,6 +658,17 @@ func (r *Repo) Add(filename string) error {
 	return nil
 }
 
+// CommitAll Commit all changes using git command
+func (r *Repo) CommitAll(msg string) ([]string, error) {
```

Comment:
Is there a reason `Commit()` doesn't suit our needs here?
If it's the `Author` field, you can push those values into consts instead.

Reply:
I resolved this to Add()ing only the release notes files. The reason I was adding a new function was because the Commit functions does not take arguments besides the message.